_data/packages.yml

```diff
@@ -874,7 +911,7 @@
     - name:
       github_username: khynder
   archive: '[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.8214655.svg)](https://doi.org/10.5281/zenodo.8214655)'
-  version_accepted: v0.2.36
+  version_accepted: v [0.2.36](https://github.com/katoss/cardsort/releases/tag/v0.2.36)
```

Comment:
Example of linking inconsistency.

Reply:
ok i'll tackle the release items later (or i'll ask the person who's starting soon to work on it!). we may just want a validator for now to clean up any additional links.